test/unit/api/ws/workers/peersUpdateRules.js

```diff
@@ -119,7 +114,7 @@ describe('PeersUpdateRules', function () {
 		it('should return an error from server when invoked with valid arguments and received error code', function (done) {
 			peersUpdateRules.slaveToMasterSender.send.restore();
```

Comment:
These restores can be removed now that we have the global sandbox and `afterEach` hook.